utilities/plugins/openai_plugin.py

```
@@ -62,7 +62,8 @@ def request_http(self, endpoint: str, query: dict[str, str], extra_param: dict[s
         data = self._construct_request_data(endpoint, query, extra_param)
         try:
             url = f"{self.host}{endpoint}"
-            response = requests.post(url, headers=headers, json=data, verify=False)
+            LOGGER.info(f"Sending request to {url} with timeout=1200s")
+            response = requests.post(url, headers=headers, json=data, verify=False, timeout=(60, 1200))
```

Ruff (0.15.12)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)